example-function.js

```diff
@@ -0,0 +1,20 @@
+function displayStatus() {
+  var appElement = document.querySelector("#app");
+  appElement.innerHTML = '';
+
```

Comment:
Lines 5-18 are very repetitive, lets extract the code into a function for re-usability.
Suggestion:
```
function appendStatusLineToElement(text, element) {
    //do stuff...
}
```